src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+2.33.1
+++++++
+
+**Compute**
+
+* Hotfix: Fix #21224: Fix the issue that VMCustomization is not enabled (#21241)
```

Comment:
```suggestion
* Hotfix: Fix the issue that VMCustomization is not enabled (#21241)
```

Reply:
It should be ok to include issue ID that is fixed. The issue ID is useful especially when someone is tracking this issue.